index.js

```diff
@@ -12,6 +12,7 @@ var PLACE_HOLDER_REGEXP = new RegExp('"@__(F|R|D|M|S)-' + UID + '-(\\d+)__@"', '
 
 var IS_NATIVE_CODE_REGEXP = /\{\s*\[native code\]\s*\}/g;
 var IS_PURE_FUNCTION = /function.*?\(/;
+var IS_ARROW_FUNCTION = /.+?=>.+?/;
```

Comment:
Why did you use `+?` not `*?`? I prefer `*?` like the above line if you don't have any reasons.

Reply:
Great catch! Yes, *? is much better. Can you make the change or do you want me to do it?

Reply:
Please make the change into your branch.

Reply:
Done